src/shared/components/case-viewer/case-viewer.component.spec.ts

```diff
@@ -452,7 +452,7 @@ let caseNotifier: any;
 let navigationNotifierService: NavigationNotifierService;
 let errorNotifierService: ErrorNotifierService;
 
-describe('CaseViewerComponent', () => {
+xdescribe('CaseViewerComponent', () => {
```

Comment:
These should be replaced by other tests or removed in the future.

Reply:
Agree

Reply:
Agree with @adnanakgun, although if the tests add value I'd push for changing them to reflect using the new material tab component.